docs/docs/artifact-store.md

```diff
@@ -4,39 +4,38 @@ title: Artifact Store
 
 # Artifact Store
 
-Deployments inside a distributed system need a location, where application specific resources can be found.
-In Marathon we call this place the artifact store.
+Deployments inside a distributed system need a location where application-specific resources can be found. In Marathon, we call this place the artifact store.
 
 
 ## Usage
 
-The use of the Marathon Artifact Store functionality is completely optional.
-All functionality and goals can be accomplished manually as well.
-Marathon tries to simplify several use cases.
+Use of the Marathon artifact store is completely optional: you can create all of the artifact atore functionality manually as well. The Marathon artifact store simplifies several use cases.
```

Comment:
The main use case: artifacts needed for a deployment have to be available in the cluster. The artifact store could be that place.